SCOPE.md

```diff
@@ -0,0 +1,32 @@
+# AsyncAPI Scope
+
+AsyncAPI is a specification for defining message-driven APIs. We're on a mission to standardize message-based communication and increase interoperability of the different types of systems out there.
+
+## Scope
+
+The following is a list of features the AsyncAPI specification and its official tooling should offer:
+
+- Provide enough information to connect and consume message-driven APIs. An AsyncAPI document should provide all your software needs to connect to a given API and start publishing and/or consuming messages from it.
```

Comment:
"all your software needs" - as a specification, it isn't providing software. It is providing a description of the existing async APIs available for consumption, right?

Reply:
Maybe like this?

An AsyncAPI document should provide **everything** your software needs to connect to a given API and start publishing and/or consuming messages from it.

It doesn't say it provides software. It provides all the information your software will need to connect and consume (maybe should be like this?). In any case, AsyncAPI also provides tooling for that purpose so it offers software, not just the spec.